src/interfaces.ts

```diff
@@ -70,4 +80,26 @@ export interface ArticleType {
       name: string;
     },
   ];
+  notes: [
```

Comment:
Er disse nye feltene i bruk noe sted?

Reply:
De er brukt i LearningResourcePanels.jsx og TopicArticleAccordionPanels.jsx som props i de individuelle panelene. [TopicArticle](https://i.imgur.com/vma9A3Q.png), [LearningResourceArticle](https://i.imgur.com/GHRGbj8.png)

Reply:
Men er de brukt eller er finnes de bare i objektet?
Er ikke så farlig om de er her uansett, men tenker det ikke er nødvendig å utvide alle typene til å inkludere masse ting vi ikke bruker.
Både pga støy, men også pga da brekker ikke ting om vi feks bestemmer oss for at et ubrukt felt skal hete noe annet eller endres.

Reply:
Et kjapt søk viser at ihvertfall **competences** og **status** blir brukt, men det er i .jsx filer så ikke så alt for farlig for akkurat denne definisjonen om de er her eller ikke. Velger å la de stå.